### PR TITLE
tweak resnet variable name

### LINKED ISSUE #67: Error running ResNet50 - Release 0.2 - incorrect argument label in call (have 'logits:labels:', expected 'logits:oneHotLabels:')
Screen Shot 2019-03-20 at 21 39 16

simple fix will submit PR.
```swift
return softmaxCrossEntropy(logits: logits, labels: oneHotLabels) -> 

return softmaxCrossEntropy(logits: logits, oneHotLabels: oneHotLabels)
```

### PR DESCRIPTION
**Pull request by brettkoonce**
labels --> oneHotLabels

Fixes #67.

### CODE REVIEW

**Comment by rxwei** [comment]
Thanks!
